.claude/rules/no-absolute-paths.md

```
@@ -0,0 +1,42 @@
+# リポジトリ内ファイルへの絶対パス記載禁止ルール
+
+## ルール
+
+コミット対象のファイル（ドキュメント、設定ファイル、コードのコメント等）に
+ユーザー名を含む絶対パスを記載してはならない。
+
+**禁止例**:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)